### PR TITLE
FCCEUD-52: Add context variables to assemblies

### PR DESCRIPTION
**Pull request by theashiot**
* Add context variable to assemblies

* Add header and footer for declaring and undeclaring parent context

### CODE REVIEW

**Comment by theashiot** [comment]
Hi @redhataccess/pantheon-developers can you please check this PR for technical correctness?

many thanks,

ashwin

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-configuring-repository.adoc [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 
+include::../attributes.adoc[]
+
 [id="configuring_repository_{context}"]
 = Configuring your repository
 
+:context: assembly-configuring-repository
+
 [role="_abstract"]
+
```
Comment (benradey) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-configuring-repository.adoc:11]: I'm not entirely sure but I think this blank line might introduce an issue in extracting the abstract here.

**Review by benradey — COMMENTED** [review]
I didn't see any content changes so this looks good from that perspective - I did see one syntax issue that I'm not sure about that I commented on. Blank lines can be a bit insidious so unless there's a reason you need it, then I recommend removing it for safety's sake.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-configuring-repository.adoc (hunk 1 above)
Comment (theashiot) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-configuring-repository.adoc:11]: Oh, yes, thanks for catching this. I'll remove the blank line.

**Comment by theashiot** [comment]
I have made the changes mentioned by @benradey and also added [role="_abstract"] to assemblies that did not have it.

Ready for peer review!

best,

ashwin

**Comment by stoobie** [comment]
@theashiot LGTM! @redhataccess/pantheon-developers Please merge.

**Review by stoobie — APPROVED** [review]
LGTM

**Review by anarang — APPROVED** [review]
Approving this PR as it looks good based on the earlier comments.

**Pull request merged**